Composer/packages/client/src/components/ProjectTree/index.tsx

```diff
@@ -163,18 +164,19 @@ export const ProjectTree: React.FC<IProjectTreeProps> = props => {
       onResizeStop={handleResize}
     >
       <div className="ProjectTree" css={root} data-testid="ProjectTree">
-        <SearchBox
-          ariaLabel={formatMessage('Type dialog name')}
-          placeholder={formatMessage('Filter Dialog')}
-          styles={searchBox}
-          onChange={onFilter}
-          iconProps={{ iconName: 'Filter' }}
-          underlined
-        />
-        <div
-          aria-live={'polite'}
-          aria-label={formatMessage(
-            `{
+        <FocusZone direction={FocusZoneDirection.vertical} isCircularNavigation={true}>
```

Comment:
This is exactly what I was trying to make work, and I think that "direction" prop is what I was missing before. Good job finding that.